### ISSUE TITLE
Large loss in yolov3 darknet finetuning 

### ISSUE THREAD

**Issue by BloomBabe** [qqwweee/keras-yolo3#752]
Hello, I downloaded the original weights yolov3 darknet, converted them to format .h5, evalute them on coco_eval, and got good results. Then I decided to start training this model at a very low learning rate, for example, 1e-9, and as a result, I get very large values ​​of the loss function (~ 10 total). Since the model has shown good results on evalution, high values ​​of the loss function can only be associated with the calculation of losses. I have a question, what is the reason for this, is it possible to fix it somehow, or if not, are there other tensorflow repositories where the original model shows the original low values ​​of the loss function for yolov3 darknet?
